blob/azureblob/example_test.go

```diff
@@ -17,82 +17,160 @@ package azureblob_test
 import (
 	"context"
 	"io"
+	"log"
 
 	"github.com/Azure/azure-storage-blob-go/azblob"
 	"gocloud.dev/blob"
 	"gocloud.dev/blob/azureblob"
 )
 
-func Example() {
-	ctx := context.Background()
+func openAzureBucket(ctx context.Context) (*blob.Bucket, error) {
```

Comment:
Sorry, but can you copy/paste this into each of the examples? It's the most interesting code for a user in the examples, and I don't think godoc will show it at all if it's in a helper.

Reply:
Sure, done.